src/Symfony/Component/Workflow/Event/Event.php

```diff
@@ -37,16 +37,25 @@ public function __construct($subject, Marking $marking, Transition $transition)
         $this->transition = $transition;
     }
 
+    /**
+     * @return Marking
```

Comment:
So should I revert it? Or create a separate PR? For me, adding some of these pointed out some details. Sure I might see that looking at the code, but...